packages/augur-ui/src/modules/create-market/components/common.tsx

```diff
@@ -266,7 +266,7 @@ export const ExplainerBlock = (props: ExplainerBlockProps) => (
   <div className={Styles.ExplainerBlock}>
     <h2>{props.title}</h2>
     {props.subtexts.map((subtext, index) => (
-      <p key={index}>{subtext}</p>
+      <li key={index}>{subtext}</li>
```

Comment:
shouldnt you change the styles to li as well

Reply:
I did change the style,

Reply:
yea i missed that. but don't think all explainer blocks are supposed to be bulleted?